<launch>

	<node pkg="kkanbu_sensor" type="kkanbu_merge" name="kkanbu_merge" >
    </node>

	<node pkg="kkanbu_sensor" type="kkanbu_camera" name="kkanbu_camera" output="screen">
    </node>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find kkanbu_sensor)/rviz/a.rviz"> 
    </node>
</launch>
